spec/models/post_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(author: User.new(name: 'Wale', posts_counter: 0), title: 'Work in progress') }
+  before { subject.save }
+
+  test_str = "ybisscougiqwfcw7ioubmgcoihkbshpjphsdhplksbjgdugubvhsdvdcygkbjopjpbjh
+        ihijldchiubvajstfhjdscoi[k;jhuksdclkn,j hoil nvhuldp0ipkoiyuidsf6usfopjpguiguvhjvkhss
+        cjpcudchidiogougjduigddkudcijbjcknkln;cjiildcj,bdbcygukvhvdcudgoiucjbsdvcyfsye5fgkjdc
+        nlkohkldviodhvln.dcidhoinkldnvihdiohvilndknvidhidkdlnviodhcoindck.n.kdcokjbskxjbkjbax
+        lnalxnknlnsclnllllllllllllllllaknxjlabxkblboihlnklnnklnlkanxjbagyxgjmxghvcchxgcghavxjgv
+        bkxkhbaxgiuuga axnghvhjavxkjbmam xjhbkjxhbkjhvxxjgvjvkabxlnkjbaxhvavwukbbsckjsbcjbscbbkjsb
+        kubkbbsaxkjksgkvsxkuhsolxb,jslkxhlsbkjhulkhskkx,bsxjgixysx lksj;cjpohgcaxvkaxakxjjbakjxbhjvjaxh
+        khvksxjhhkgkskhklsxsxiugsucouugosxkslxnjbkbaxgch ajxgakvkhsbksixuusgxisxhhvsxsxygsixgsx
+        kuubkscjnkscschvsciuscibsckjsbksvjsbksbsvcsvgvhgcaxbakxbkjaxhbbajvjaxhvigisxvgsjusgusx
+        jgjgvgjsxkjnskxksglsliihslsxlihsxysisjxjhgsjxhbsjxvghscxhgsxhsxgchsxusgxugusxysxusuxyyggs
+        kukusxgugsxguihgisxyfsysxhgchassxvjsxsjvjsxxsiuisxsjkkxiusxsufusxsuxygyusxyfsutxsxvjsxj
```

Comment:
Your tests are failing:
<img width="516" alt="Screen Shot 2022-07-16 at 5 30 44 AM" src="https://user-images.githubusercontent.com/69011963/179327212-4455cbed-3d5e-43c7-94e2-d15c55529bfd.png">

- [ ] Please make sure your all tests are running correctly. Also, it would be great idea if you could add screenshot of your tests are passing in Pull Request.